samples/cpp/benchmark_app/inputs_filling.cpp

```diff
@@ -571,7 +571,7 @@ ov::Tensor get_random_tensor(const std::pair<std::string, benchmark_app::InputIn
     } else if (type == ov::element::f64) {
         return create_tensor_random<double, double>(inputInfo.second);
     } else if (type == ov::element::f16) {
-        return create_tensor_random<ov::float16, short>(inputInfo.second);
+        return create_tensor_random<short, short>(inputInfo.second);
```

Comment:
It doesn't matter much because it is random, but should both template values be `ov::float16`?

Reply:
The reasons is `uniformDistribution<T2>` won't compile with `ov::float16`.

Never mind